example/src/index.ts

```diff
@@ -12,15 +12,17 @@ async function main() {
 
   await photon.connect()
 
-  const nexusPrisma = nexusPrismaPlugin({
+  const nexusPrisma = NexusPrisma.nexusPrismaPlugin({
     photon: ctx => ctx.photon,
   })
 
-  const schema: any = makeSchema({
+  NexusPrisma.generateTypes()
```

Comment:
Temporary until prisma/nexus#200

Reply:
Why can't we generate the types when calling `nexusPrismaPlugin`, just like `nexus` generate types when calling `makeSchema` when not in production?

Reply:
Done!